package.json

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
Do we really need another package.json if one is already present in application. It will create a merge conflict in the application because the same file is present in backend branch also. For the application Integration we can see it today.

Reply:
removed package files